twill-common/src/main/java/org/apache/twill/internal/Constants.java

```diff
@@ -78,6 +78,15 @@ private Files() {
     }
   }
 
+  /**
+   * Constants for twill environment variable names.
+   */
+  public static final class Environments {
```

Comment:
They are initially used in EnvKeys to pass system env variables. But with the use of TwillRuntimeSpecification, they are no longer used as keys. They are used as constants in configuration or error messages so I move them to Constants class